build-tools/automation/build.groovy

```diff
@@ -255,7 +253,7 @@ timestamps {
 
             if (!isPr) {
                 if (isCommercial) {
```

Comment:
since the if case is now a no-opt, just handle the non-commercial (OSS) case w/ if (!isCommercial)

Reply:
Ahh yeah makes sense, this is what multitasking gets me 😄